lib/mpi/messages/find_profile_message_with_address.rb

```diff
@@ -0,0 +1,71 @@
+# frozen_string_literal: true
+
+module MPI
+  module Messages
+    class FindProfileMessageWithAddress < FindProfileMessage
```

Comment:
I might recommend rearranging this a bit, I think it's a bit confusing to inherit from `FindProfileMessage` and the gymnastics you have to do to make some of the existing functions work. Here's my recommendation:
- No longer inherit from `FindProfileMessage`
- Include `FindProfileMessageHelpers`
- Keep `initialize`, `required_fields_present?` and `build_parameter_list` in `FindProfileMessageWithAddress`
- Move `build_address` (and the 'build' functions from `FindProfileMessage`, like `build_living_subject_name`, `build_living_subject_birth_time`, etc) into `FindProfileMessageHelpers`

Reply:
> and the gymnastics you have to do...

yeah, I was trying to take advantage of most of the existing code, since this class mostly just adds an xml stanza for address data while omitting the SSN one, as well as adapting `required_fields_present?` to that fact.

I'll play around with these suggestions and see if it makes things a little cleaner

Reply:
Added this through a number of subsequent commits, so resolving this, can follow up in any new comments